book/display.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC, abstractmethod
+
+from book.book import Book
+
+
+class Display(ABC):
+    @abstractmethod
+    def display(self, book: Book) -> None:
+        pass
+
+
+class ConsoleDisplay(Display):
+    def display(self, book: Book) -> None:
+        print(book.content)
+
+
+class ReverseDisplay(Display):
```

Comment:
The same logic applies to `ReverseDisplay`. It should be renamed to `DisplayReverse`. The class name should indicate that it displays content in reverse order, with "Display" (what it does) coming first, followed by "Reverse" (how it does it).